src/Rimu.jl

```diff
@@ -38,18 +35,19 @@ export TimeStepStrategy, ConstantTimeStep, OvershootControl
 export threadedWorkingMemory, localpart, walkernumber
 export RimuIO
 
+include("report.jl")
 include("strategies_and_params.jl") # type defs and helpers
 include("helpers.jl")               # non MPI-dependent helper functions
 include("fciqmc_col.jl")            # third level
 include("apply_memory_noise.jl")
 include("fciqmc_step.jl")           # second level
-include("norm_project.jl")
-include("lomc.jl")                  # top level
+include("update_dvec.jl")
+include("QMCState.jl")              # top level
```

Comment:
May I suggest to rename the file "QMCState.jl" to "lomc.jl"? The reason is that files with capital letters are all modules and it would be nice to stick with this rule. I'm open to suggestions for better names, though.